plugins/wacom-raw/fu-wacom-aes-device.c

```diff
@@ -220,6 +220,22 @@ fu_wacom_aes_device_write_block(FuWacomAesDevice *self,
 	return TRUE;
 }
 
+static gboolean
+fu_wacom_aes_device_finalize_firmware(FuDevice *device, GError **error)
```

Comment:
shouldn't this be fu_wacom_aes_device_detach() i.e. implementing the FuDevice->detach() vfunc?

Reply:
Sure, that's possible.